base/Dockerfile

```diff
@@ -15,14 +16,33 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 WORKDIR /var/www
 RUN git clone https://github.com/NYPL-Simplified/circulation.git
 WORKDIR circulation
-RUN git checkout tags/$version
-RUN /bin/bash -c 'git config submodule.core.url https://github.com/NYPL-Simplified/server_core.git && git submodule update --init --recursive'
+
+# Assume we're building for a tagged commit of circulation, but also
+# allow for branches or straight-up SHAs if --build-args tags=f is used
+# at build time.
+RUN /bin/bash -c 'if [[ "$tags" == "t" ]];\
```

Comment:
This `RUN` command and the build arg on line 5 are the only code changes. Everything else is a whitespace or documentation change.